databricks/koalas/frame.py

```diff
@@ -9122,6 +9122,179 @@ def gen_new_column_labels_entry(column_labels_entry):
         else:
             return DataFrame(internal)
 
+    def rename_axis(
+        self,
+        mapper: Optional[Any] = None,
+        index: Optional[Any] = None,
+        columns: Optional[Any] = None,
+        axis: Optional[Union[int, str]] = 0,
+        inplace: Optional[bool] = False,
+    ) -> Optional["DataFrame"]:
+        """
+        Set the name of the axis for the index or columns.
+
+        Parameters
+        ----------
+        mapper : scalar, list-like, optional
+            A scalar, list-like, dict-like or functions transformations to
+            apply to the axis name attribute.
```

Comment:
Maybe this explanation for `mapper` is not correct?

In the pandas latest docs, they say:

```
Value to set the axis name attribute.
```

Reply:
Yes, I should have precised that pandas does not support using a dict or a function as `mapper`. They say :  
`However, if mapper is dict-like or a function, it will use the deprecated behavior of modifying the axis labels.`
So, I am not sure this is the correct behavior, that is why I added the possibility to use dict / function as `mapper`, and therefore updated the docs.
 I am interested in your opinion on this !